examples/api/ablation_paths.py

```diff
@@ -4,6 +4,7 @@
 
 This example shows how to use the plugin Ablation Paths.
 Note that other plugins use the same interfaces and can be used in the same fashion.
+To use the API commands the according record example needs to be run in order create the run to analyze.
```

Comment:
To use the API commands in this example, we assume the run data to be already present in the mentioned folder. Please ensure that you have the run data located at the corresponding path.